packages/node-build-scripts/package.json

```diff
@@ -6,6 +6,7 @@
     "test": "exit 0"
   },
   "bin": {
+    "add-js-version": "./add-js-version.sh",
```

Comment:
How the add-js-version ends in node_modules/.bin/add-js-version.sh ?  Is it in the yarn install of this package ?

Reply:
Yes, the `yarn install` 👍

Reply:
👍